ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -60,6 +66,18 @@ public class RNSSplitViewScreenController: UIViewController {
     return self.splitViewController is RNSSplitViewHostController
   }
 
+  ///
+  /// @brief Determines whether an SplitView animated transition is currently running
+  ///
+  /// Used to differentiate favor frames from the presentation layer over view's frame .
+  ///
+  /// @return true if the transition is running, false otherwise.
+  ///
+  @objc
+  public func isTransitionInProgress() -> Bool {
+    return transitionInProgress
+  }
```

Comment:
`isViewSizeTransitionInProgress` & if you have a dedicated `ViewSizeTransitionTrackingState` struct then just `return viewSizeTransitionTrackingState != nil`.

Reply:
https://github.com/software-mansion/react-native-screens/pull/3176